packages/common/src/Widget.ts

```diff
@@ -652,10 +652,15 @@ export abstract class Widget extends PropertyExt {
         });
     }
 
-    lazyRender = debounce(function (debouncedCallback?: (w: Widget) => void) {
+    private _lazyRender = debounce(function (debouncedCallback?: (w: Widget) => void) {
         this.render(debouncedCallback);
     }, 100);
 
+    lazyRender(): this {
+        this._lazyRender();
+        return this;
+    }
```

Comment:
@GordonSmith is this what you had in mind?

Reply:
Yup.